command/scripting.go

```diff
@@ -436,3 +523,27 @@ func (c *scriptingExec) Execute(ctx context.Context, comm client.Communicator, l
 
 	return catcher.Resolve()
 }
+
+func (c *scriptingExec) reportTestResults(ctx context.Context, comm client.Communicator, logger client.LoggerProducer, conf *model.TaskConfig, report io.Reader) error {
+	switch c.Harness {
+	case "go", "golang":
+		// kim: TODO: I don't actually know if the test name is consequential,
```

Comment:
I just specified a dummy name for `(model.TestLog).Name`. Don't know if this is valid or if the name even matters (it seems like it doesn't).